app/static/js/share/share.js

```diff
@@ -36,6 +37,18 @@ $(function () {
     $(".nav li").removeClass("active");
     $('a[href=".' + this.location.pathname + '"]').parents('li,ul').addClass('active');
 
+    // Prevents user from typing non numeric keys
+    $("#user-phone-field").keypress(function (key) {
```

Comment:
Instead of using the jquery selector for the field, you can use the variable `userPhone` as you have initialized it in line 16.